content/tutorials/get-started/get-started-with-rippleapi-for-javascript.md

```diff
@@ -152,6 +185,7 @@ When a Promise finishes with its asynchronous operations, the Promise runs the c
 
 Finally, we have more new ECMAScript 6 syntax - an [arrow function](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions). Arrow functions are a shorter way of defining anonymous functions. This is convenient for defining lots of one-off functions as callbacks. The syntax `()=> {...}` is mostly equivalent to `function() {...}`. If you want an anonymous function with one parameter, you can use a syntax like `info => {...}` instead, which is almost the same as `function(info) {...}` syntax.
```

Comment:
I think we can remove the mention of ECMAScript 6. It is not new anymore (it's been around since 2015) and most JS developers will be familiar with it. Suggested replacement:

> The example uses [arrow functions](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions), a shorter way of defining anonymous functions.